nms/app/packages/magmalte/app/views/subscriber/SubscriberTable.js

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+import type {ActionQuery} from '../../components/ActionTable';
+import type {WithAlert} from '@fbcnms/ui/components/Alert/withAlert';
+
+import ActionTable from '../../components/ActionTable';
+import NetworkContext from '../../components/context/NetworkContext';
+import React from 'react';
+import SubscriberContext from '../../components/context/SubscriberContext';
+import nullthrows from '@fbcnms/util/nullthrows';
+import withAlert from '@fbcnms/ui/components/Alert/withAlert';
+
+import {DEFAULT_PAGE_SIZE} from '../../views/subscriber/SubscriberUtils';
+import {JsonDialog} from './SubscriberOverview';
+import {RenderLink} from './SubscriberOverview';
+import {handleSubscriberQuery} from '../../state/lte/SubscriberState';
```

Comment:
One thing which we might have to fix in future PR is to enable search through the subscriber table. The query can contain more than pageSize. So we might have to invoke other queries on subscriber API involving searching through IMSI or MSISDN